lib/CL/pocl_mem_management.h

```diff
@@ -111,6 +116,19 @@ pocl_convert_to_subbuffer_migrations (pocl_buffer_migration_info *buffer_usage,
 
 #endif
 
+/**
+ * Get the device memory pointer of the supplied pocl argument.
+ *
+ * \param global_mem_id [in] This is needed to get the device specific pointer.
+ * \return NULL if arg->value is NULL and otherwise the requested pointer.
+ */
+POCL_EXPORT
+void *pocl_cpu_get_ptr (struct pocl_argument *arg, unsigned global_mem_id);
```

Comment:
I think the return value should be in its own line, but clang-format gets confused by POCL_EXPORT. Please fix that in the next PR.

Reply:
most of the uses of pocl_export I see look like this.

Reply:
Yeah, but our style guide is "GNU style for the C sources" and we wan tho converge towards the style guide whenever we commit. Looking at [the guide](https://www.gnu.org/prep/standards/html_node/Formatting.html#Formatting), the correct one would be maybe to have `POCL_EXPORT void` in the same line?

Reply:
I come to the same conclusion reading the guide, I've updated it.